packages/augur-ui/src/modules/portfolio/components/portfolio-view/my-bets.tsx

```diff
@@ -113,7 +184,15 @@ export const MyBets = () => {
             </span>
           </section>
         )}
-        {rows.length > 0 && rows.map(row => <Game row={row} type={SPORTS_MARKET_TYPES[selectedMarketCardType].label} />)}
+        {rows.length > 0 &&
```

Comment:
Don't need row.length > 0 if we are going to map, if you map on an empty array it won't do anything anyway so we can just do the showEvents and map